src/types/User.ts

```diff
@@ -0,0 +1,6 @@
+export interface User {
+  id?: number;
```

Comment:
Data coming from server should have some id in it. I'm not used to any id being omitted. Definitelly not asking you to rewrite here, just wanted to note